### ISSUE TITLE
Add single font to collection, not necessarily whole font-family

### ISSUE THREAD

**Issue by Pascalio** [FontManager/font-manager#221]
**Is your feature request related to a problem? Please describe.**
Very often I find that _only_ some "weight" or "variation" of a font family is interesting for a particular type of purpose; and it's not always the flagship variant, it's not always the "regular" one: it can be hidden in some "light" or so variation.

**Describe the solution you'd like**
If collections could store individual fonts, on a per-file basis, instead of only storing the family, it would be a great way to collect only those variations that interest for a particular purpose. 
From a UI perspective, dragging the font-family list item would add the whole family to the collection, while dragging one sub-variation would only add that one and ctrl-select a few and then dragging would add those selected ones only.

**Describe alternatives you've considered**
Alternatively, I just keep track of interesting font variations in a .txt file and consult it when I'm feeling lost, but I thought a font manager could be a more practical and portable way (allowing to save stripped-down collections and reinstall them on new machines)

**Additional context**
I know you have said in #22 that it was low in your priority list, but I just wanted to put forward some more keen interest in this feature!

**Comment by JerryCasiano** [comment]
> I know you have said in #22 that it was low in your priority list, but I just wanted to put forward some more keen interest in this feature!

Actually, what I said was that the complexity added by this isn't worth it in my opinion. So... very unlikely to happen unless someone steps up to do it and why I ended up closing that issue. It's actually not low priority but more like no priority.

But I've been known to change my mind quite a bit so...

We'll see maybe I'll have a look during the winter months when I'm stuck inside and bored. :-)

**Comment by amigaosz** [comment]
Hello, I have probably a stupid question. I've already looked for an answer but I couldn't find anything.

I have installed the Font Manager via Flathub/Flatpak and created some categories. Then I thought I can simply drag a font from the right pane onto the collection entry in the list left but this doesn't work. How can I add fonts to my collections? Or do I have to drag and drop the files into the appropriate collection? My thought was that a font can be part of many collections.

**Comment by JerryCasiano** [comment]
You thought right.

Unless something is broken, which is entirely possible, you should be able to drag fonts from the list into collections.

And yes a font can be present in many collections.

I'll see if I can duplicate this later tonight and get back to you.

**Comment by amigaosz** [comment]
Thank you very much for your fast answer! :)

I've uninstalled the Flathub version and installed Font Manager via **apt** now and here I can move the fonts as expected to the self created collections. It also looks like the Flathub version has not recognized all user installed fonts, now I see even more of my self installed fonts.

I don't know why but I've had several times some small problems with software installed via Flathub in the past. I always thought this is my fault because I broke my system somehow but now I have a brand new laptop and installed Xubuntu 22.04 since a week.

However: This is fine for me, it works now like a charm and I love this wonderful piece of software! Thank you so much for this font manager! :)

**Comment by JerryCasiano** [comment]
@amigaosz 

Yes, unfortunately the Flatpak tends to display certain issues at times that are not present in a normal installation.

Some we can fix, but some, like this one, are probably not something we can fix on our end.

In any case, thanks for the update and I'm glad it's working for you now.

**Comment by JerryCasiano** [comment]
I'm closing this out.

It appears I'm never going to be bored enough to implement this...
